src/components/common/core.tsx

```diff
@@ -2,6 +2,7 @@ import styled from "styled-components";
 import { Flex } from "@rebass/grid";
 import Dropdown, { Option } from "react-dropdown";
 import JSBI from "jsbi";
+import BigNumber from "bignumber.js";
```

Comment:
Hmm, not sure about this one, not great that we're using 2 bignum libraries

Reply:
We need BigNumber to handle floats from user inputs. 
We might be able to remove JSBI from Lens as the new changes for wavelet-client should allow us to pass in strings.

Reply:
Interesting, JSBI is also a larger dependency than bignumber.js. How big of a refactor do you estimate ripping out JSBI is for lens? Although I guess if it's a dep for wavelet-client, we're not going to be seeing much space saving

Reply:
Doesn't look trivial, a quick search yielded about 55 JSBI calls. It worth it to simplify Lens, but it'll take some time and has a risk of bugs.

Reply:
Ok, sure, lets just merge this one then